test/unit_tests/braket/experimental/autoqasm/test_api.py

```diff
@@ -917,3 +917,20 @@ def main():
 
     with pytest.raises(errors.AutoQasmTypeError):
         main()
+
+
+def test_empty_decorator_parentheses():
```

Comment:
Cool! Not sure why I got errors when I tried, my solution looked the same. Must have been something silly!